marlowe-chain-sync/app/Main.hs

```diff
@@ -0,0 +1,86 @@
+module Main where
```

Comment:
Eventually, we should probably move `main :: IO ()` to the CLI component for Marlowe runtime.

Reply:
I understand that no action required on this for now, but this highlights the need for us to discuss and get on the same page regarding the plan for the CLI, as this doesn't align with what I had imagined. In my mind, the CLi does not control the lifecycle of runtime processes, it simply is an interface that connects to an already running runtime via IPC to issue commands and queries.

Not saying one way or another that I agree or disagree, merely drawing attention to the fact that there is currently a lack of alignment here.